test/integration/index.spec.ts

```diff
@@ -1426,75 +1425,6 @@ describe('Worker', () => {
 		await expect(ctx.flush(ctx.session)).rejects.toThrowError();
 	});
 
-	it('should not re-enqueue requests after execute failure', async () => {
-		const typeContract = ctx.worker.typeContracts['card@1.0.0'];
-		assert(typeContract);
-
-		await ctx.worker.insertCard(
-			ctx.logContext,
-			ctx.session,
-			ctx.worker.typeContracts['action-request@1.0.0'],
-			{},
-			{
-				data: {
-					action: 'action-create-card@1.0.0',
-					context: ctx.logContext,
-					card: typeContract.id,
-					type: typeContract.type,
-					actor: ctx.adminUserId,
-					epoch: new Date().valueOf(),
-					input: {
-						id: typeContract.id,
-					},
-					timestamp: new Date().toISOString(),
-					arguments: {
-						reason: null,
-						properties: {
-							slug: 'foo',
-							version: '1.0.0',
-							data: {
-								foo: 'bar',
-							},
-						},
-					},
-				},
-			},
-		);
-
-		const enqueuedRequest1 = await ctx.dequeue();
-		assert(enqueuedRequest1);
-
-		await ctx.kernel.insertContract(ctx.logContext, ctx.session, {
```

Comment:
Here's it's generating a new `card` to mock what the worker would do if consuming the task.